addons/l10n_fr_account/models/base_document_layout.py

```diff
@@ -0,0 +1,15 @@
+from markupsafe import Markup
+from odoo import api, fields, models, _
+
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_report_footer(self):
+        # Override to add the company.registry for FR companies
+        if not (self.env.company.company_registry and self.env.company.country_code == 'FR'):
+            return super()._default_report_footer()
+        return super()._default_report_footer() + Markup('<br/>%s') % _('SIRET: %s', self.env.company.company_registry)
```

Comment:
Small nitpicking but usually i see more something like this but it's the same so feel free to ignore: 
```
footer = super()._default_report_footer()
if self.env.company.company_registry or self.env.company.country_code != 'FR'
    footer + Markup('<br/>%s') % _('SIRET: %s', self.env.company.company_registry)
return footer
```